lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb

```diff
@@ -0,0 +1,571 @@
+{
```

Comment:
<blockquote><s>The positional arguments of the PyMedPhys </s><a href="https://github.com/pymedphys/pymedphys/raw/c029675cd7d66b663c9ab1971b55fa38be5c38cc/lib/pymedphys/docs/lib/ref/gamma.rst" target="_blank"><s>gamma</s></a><s> specify the axes and dose of the reference and evaluation images, as well as the dose difference and distance to agreement to perform the computation. </s></blockquote><p> </p><p>This could benefit from a little more fleshing out, I think. Maybe not necessary for a first pass. Maybe for now this cell is better off saying something like:</p><p> </p><blockquote>Now, we need to extract the dosegrids as well as the x, y, and z axes from the reference and evaluation RTDOSE datasets in a form readable by the PyMedPhys gamma function: You can use <code>zyx_and_dose_from_dataset()</code> for this like so: </blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
I agree. Also, I think it's more readable.